src/components/Pagination/Pagination.scss

```diff
@@ -1,12 +1,12 @@
 .pager {
   &-limit {
     display: inline;
-    width: unset;
+    width: 80px;
```

Comment:
Does this work in mobile?

Reply:
FYI, I adjusted this a bit so this change wouldn't affect the browsers `unset` works in and added `max-width` to fix the issue in Firefox.

Reply:
Toggling between `master` and the PR, the displays in a mobile view port size are the same for Chrome and the fix works in Firefox. I can do some adjustments if you want, as the offset in `xs` pushes everything to the right on smaller viewports.